realm/realm-library/src/objectServer/java/io/realm/SyncManager.java

```diff
@@ -48,6 +48,18 @@
 public class SyncManager {
 
     /**
+     * Debugging related options.
+     */
+    @SuppressFBWarnings("MS_SHOULD_BE_FINAL")
```

Comment:
Hmm, doesn't look possible. Only possible to differ between debug/release builds and we probably don't want to disable the check for all debug builds. In that case this is probably fine